sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -47,322 +44,259 @@
-        <version>1.0.1</version>
-      </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-common</artifactId>
-        <version>1.0.1</version>
+        <version>1.0.2</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-identity</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.1</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-phonenumbers</artifactId>
-        <version>1.0.2</version>
+        <version>1.0.3</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-sms</artifactId>
-        <version>1.0.2</version>
+        <version>1.0.3</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core</artifactId>
-        <version>1.16.0</version>
+        <version>1.17.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-amqp</artifactId>
-        <version>2.0.5</version>
+        <version>2.2.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-http-netty</artifactId>
-        <version>1.9.2</version>
+        <version>1.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-http-okhttp</artifactId>
-        <version>1.6.2</version>
+        <version>1.7.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-management</artifactId>
-        <version>1.2.2</version>
+        <version>1.3.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-gson</artifactId>
-        <version>1.1.2</version>
+        <version>1.1.3</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-jackson</artifactId>
-        <version>1.2.3</version>
-      </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-test</artifactId>
-        <version>1.6.2</version>
+        <version>1.2.4</version>
       </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-tracing-opentelemetry</artifactId>
-        <version>1.0.0-beta.10</version>
-      </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-cosmos</artifactId>
-        <version>4.15.0</version>
-      </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-data-appconfiguration</artifactId>
```

Comment:
>azure-data-appconfiguration [](http://example.com/codeflow?start=20&length=27)

This was getting removed because the version_client.txt version was dependening on core 1.18. I regenerated with the overload list. Thanks for catching.